lib/text/ui_text_displayer.js

```diff
@@ -417,11 +417,33 @@ shaka.text.UITextDisplayer = class {
     regionElement.id = 'shaka-text-region---' + regionId;
     regionElement.classList.add('shaka-text-region');
 
-    regionElement.style.height = region.height + heightUnit;
-    regionElement.style.width = region.width + widthUnit;
     regionElement.style.position = 'absolute';
-    regionElement.style.top = region.viewportAnchorY + viewportAnchorUnit;
-    regionElement.style.left = region.viewportAnchorX + viewportAnchorUnit;
+
+    const linesUnit = shaka.text.CueRegion.units.LINES;
+    if (region.heightUnits === linesUnit && region.widthUnits === linesUnit) {
+      // from https://dvcs.w3.org/hg/text-tracks/raw-file/default/608toVTT/608toVTT.html#caption-window-size
+      regionElement.style.height = region.height * 5.33 + '%';
+      regionElement.style.width = region.width * 1.9 + '%';
```

Comment:
Currently, assuming the video's aspect ratio is 16:9. 708 captions only know about 4:3 and 16:9 video. Happy to do a check here to figure out the correct multiple here but need a pointer to how to figure it out.

Reply:
I have added a commit to this PR to calculate the aspect ratio, you just have to use the variable `this.aspectRatio_`